pages/alerts.rst

```diff
@@ -312,6 +314,13 @@ Configuration Color
 Custom Message
     The message that will be sent to Slack.  The data described above can be used in this template.
 
+Backlog Message Items (optional)
+    The custom message has a data point named `backlog <http://localhost:8000/pages/alerts.html?highlight=alerts#backlog>`__, which gives
+    the preview of the messages that led to a slack notification.
```

Comment:
I don't think we need to describe here what the message backlog is.  That should be discussed elsewhere in the Alert documentation.